docs/_v5_3/5_asynkronitet.md

```diff
@@ -30,13 +30,14 @@ Task<IEnumerable<Person>> personer = register.HentPersonerAsynkront(personidenti
     Informasjonsbehov.SikkerDigitalPost ,
     Informasjonsbehov.VarslingsStatus
     );
-//Vent på at tasken blir ferdig..
 
+//Vent på at tasken blir ferdig..
```

Comment:
Skrivefeil! Jeg irriterer meg grønn over andre som bare skriver to punktum her. Setter inn ellipse :)